dev-sync.py

```diff
@@ -374,11 +142,6 @@ def group_from_github(ctx, token, project, from_ref, to_ref, status, linear, dry
 	ctx.obj['from_ref'] = from_ref
 	ctx.obj['status'] = status
 
-	if status in PRODUCTBOARD_TO_ENV:
-		ctx.obj['environment'] = PRODUCTBOARD_TO_ENV[status]
-	else:
-		ctx.obj['environment'] = 'Development'
-
```

Comment:
I don't think you should remove this, but rather add a way to specify the environment from the calling action ?

Reply:
yes seems to be used in the other func but don't understand why this and not using status directly 🤷